_posts/2020-06-03-virtual-workshop-recap.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Virtual US-RSE Workshop Recap
+tags: [workshop]
+---
+
+by Jordan Perr-Sauer, Vanessa Sochat, Ian Cosden, Charles Ferenbaugh, Daniel S. Katz, Sandra Gesing
+
+Given recent current events around the novel coronavirus outbreak, many individuals and organizations have been forced to adapt. For the US Research Software Engineer Association (US-RSE), this meant the unfortunate postponement of an in-person community building workshop that was to take place April 23-24th, 2020. In order to keep up the community's strong momentum, we decided to host a [virtual workshop](https://us-rse.org/events/2020/2020-04-virtual-workshop) that could be attended from the comfort and safety of our own homes. While not aiming to replace the in-person workshop, this virtual workshop format came with some benefits of its own, such as additional participation by those who would not have been able to attend the in-person workshop, reduced environmental impact from not having to travel, and less disruption in the lives of attendees.
+
+The virtual workshop was well attended, with 30-90 participants in each session. The overwhelming majority of participants were affiliated with academic institutions in the United States. To avoid so-called “Zoom fatigue,” the workshop was planned in three sessions of 1.5 hours each, held over two days. The three sessions were called: “The RSE Landscape”, “Technical Talks”, and “Next Steps for the US-RSE Association”, which was a breakout and discussion session. Recordings of [session 1](https://www.youtube.com/watch?v=TaBMeBN7g_4) and [session 2](https://www.youtube.com/watch?v=OiJd0y38b0Q) are available on Youtube.
+
+**Session 1: The RSE Landscape**
```

Comment:
I think I'd suggest an h3 or h4 tag here (eg., three or four #) but I'll look at the preview first.

Reply:
Done. I used H3